src/SFA.DAS.AssessorService.Application/Handlers/Apply/Review/WithdrawalApplicationsHandler.cs

```diff
@@ -33,8 +37,40 @@ public async Task<PaginatedList<ApplicationSummaryItem>> Handle(WithdrawalApplic
 
             var items = Mapper.Map<IEnumerable<ApplicationListItem>, IEnumerable<ApplicationSummaryItem>>(organisationApplicationsResult.PageOfResults);
 
+            if(items.Any())
+            {
+                // only want to do this once.
+                var allEnrolledVersions = await _standardRepository.GetEpaoRegisteredStandardVersions(items.First().EndPointAssessorOrganisationId);
+
+                // Get versions for each standard and determine the withdrawal application type.
+                foreach (var item in items)
+                {
+                    var allEnrolledStandardVersions = allEnrolledVersions.Where(v => v.IFateReferenceNumber == item.StandardReference).ToList();
```

Comment:
Can we just tweak this method a bit?

The comments can probably be removed I think
// Only want to do this one  - is shown by the call outside the loop
// Get Versions for each standard  - seems to be in the wrong place? or outdated?

var allEnrolledStandardVersions - maybe enrolledVersionsForStandard?  the "All" bit I think confuses with the previous call